app/assets/javascripts/transcripts.js

```diff
@@ -65,6 +65,32 @@ $(document).ready(function(){
       new ImageCrop
     }, 100 );
   };
+
+  $('#select-all').click(function () {
+    $('.check').prop('checked', $(this).prop('checked'));
+  });
+
+  $('.item-list-headings').on('click', '.update-multiple', function(e) {
+    e.preventDefault();
```

Comment:
as i can remember, `return false;` at the end of this function should do the same effect and is more standardized?